batch/batch/driver/main.py

```diff
@@ -1022,13 +1023,13 @@ async def check(tx):
     CAST(COALESCE(SUM(state = 'Creating' AND cancelled), 0) AS SIGNED) AS actual_n_cancelled_creating_jobs
   FROM
   (
-    SELECT batches.user, jobs.state, jobs.cores_mcpu, jobs.inst_coll,
+    SELECT job_groups.user, jobs.state, jobs.cores_mcpu, jobs.inst_coll,
```

Comment:
We need it for an index for the scheduler / autoscaler to be fast. So since we have it already here, I didn't see the point of joining with the batches table to get that field.